spec/quality_spec.rb

```diff
@@ -191,6 +191,58 @@ def check_for_specific_pronouns(filename)
     expect(error_messages.compact).to be_well_formed
   end
 
+  it "documents all used settings" do
+    exemptions = %w(
```

Comment:
yup, as soon as this PR gets into master I was planning on making an issue linking to the list